runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/Dart.test.stg

```diff
@@ -269,7 +269,7 @@ LRListener(X) ::= <<
 class LeafListener extends TBaseListener {
   void exitE(EContext ctx) {
     if (ctx.childCount==3) {
-      stdout.write("${ctx.e(0)?.start?.text} ${ctx.e(1)?.start?.text} ${ctx.es()[0].start?.text}\n");
+      print("${ctx.e(0)?.start?.text} ${ctx.e(1)?.start?.text} ${ctx.es()[0].start?.text}");
```

Comment:
shouldn't this also be TEST_platformStdoutWrite?

Reply:
This line had a newline add the end. The change is to replace the 'write' with 'writeln' (print).